File: dnaToRna/DnaToRna.py
def ToRna(dna):
	
	return dna.replace("T", "U")

def main():
	dna = input("Enter a string of nucleobases: ")
	rna = ToRna(dna)
	print("The results are\n{}".format(rna)

if __name__ == '__main__':
	main()

File: dnaToRna/DnaToRna.java
import java.util.Scanner;

public class DnaToRna {

	public static String ToRna(String bases) {
		String rna;

		rna = bases.toUpperCase().replace('T', 'U');

		return rna;
	}
	
	public static void main(String[] args) {
		Scanner reader = new Scanner(System.in);
		String dna;

		System.out.println("Enter a string of nucleobases:");
		dna = reader.nextLine();

		System.out.printf("RNA Sequence:\n%s\n", ToRna(dna));
	}
}